src/Base/Support/CreatesFields.php

```diff
@@ -136,21 +135,50 @@ protected function getHeaders(string $url): array
         return $response;
     }
 
-    /**
-     * SSL is usually not valid in local environments.
-     * Disable verifications.
-     */
-    protected function streamContext()
-    {
-        if ('development' !== ($_ENV['APP_ENV'] ?? '')) {
-            return null;
-        }
-
-        return stream_context_create([
-            'ssl' => [
-                'verify_peer' => false,
-                'verify_peer_name' => false,
-            ],
-        ]);
-    }
+	protected function get_headers_curl(string $url): array
```

Comment:
En de snake_case schrijfwijze aanpassen naar camelCase voor de uniformiteit in dit bestand en de gehele plug-in. 
Ik weet dat het indruist tegen de standaarden van WordPress ;)